introduction/hardware-options/recommended-extras/camera-gimbal.md

```diff
@@ -0,0 +1,15 @@
+# Camera Gimbal
+
+ArduSub supports control and attitude atabilization for up to 3-axis gimbals. The most common gimbal type for underwater vehicles is a 1-axis tilt mount.
```

Comment:
"The most common gimbal type for underwater vehicles is a 1-axis tilt mount, since the majority of the frame has control over the others DoF."

Reply:
fixed